pkg/controller/fileintegrity/fileintegrity_controller.go

```diff
@@ -23,6 +23,143 @@ import (
 
 var log = logf.Log.WithName("controller_fileintegrity")
 
+const (
+	fileIntegrityNamespace     = "openshift-file-integrity"
+	defaultConfDataKey         = "aide.conf"
+	defaultConfigMapName       = "aide-conf"
+	workerDaemonSetName        = "aide-worker"
+	masterDaemonSetName        = "aide-master"
+	aideScriptConfigMapName    = "aide-script"
+	operatorServiceAccountName = "file-integrity-operator"
+	aideScriptPath             = "/scripts/aide.sh"
+)
+
+var aideScript = `
```

Comment:
This is more of a nitpick, so feel free to do it or not. But maybe we should move the constants (scripts) to their own file. e.g. constants.go or aide_constants.go something of the sort. This way, we would have one place to edit and look for the AIDE configurations.

Reply:
Done :)